dnf-plugins-core.spec

```diff
@@ -86,6 +86,52 @@ reposync commands. Additionally provides generate_completion_cache and noroot
 passive plugins.
 %endif
 
+%package -n dnf-utils
+Conflicts:      yum-utils < 1.1.32
+Requires:       dnf >= %{dnf_lowest_compatible}
+Requires:       %{name} = %{version}-%{release}
+%if %{with python3}
+Requires:       python3-dnf-utils = %{version}-%{release}
```

Comment:
This is work-eround how to allow:
(python3-dnf, python3-dnf-utils) -> repoquery - runs with python3
(python2-dnf, python2-dnf-utils) -> repoquery - runs with python2
(python2-dnf, python2-dnf-utils, python3-dnf, python3-dnf-utils)) -> repoquery - runs with python3
The PR shadows the solution of dnf how to redirect dnf to proper python version (dnf-2, dnf-3).